### PR TITLE
Fix CO-392 live-started pending-reopen refresh

### PR DESCRIPTION
**Pull request by Kbediako**
## What

- Adds a bounded control-host refresh probe for no-run `released-pending-reopen` claims whose cached Linear workflow truth is stale but live Linear has advanced to `In Progress` / `started`.
- Routes the recovered live-started claim through normal admission, or records `provider_issue_refresh_start_blocked:max_concurrency` with refreshed live metadata when state or global admission is capped.
- Preserves same-issue occupancy protection and fresh-discovery state-slot reservation while still refreshing stale pending-reopen metadata.
- Adds the CO-392 docs packet, task mirrors, registry entries, and focused regression coverage for the CO-379 shape.

## Why

CO-379 could remain indefinitely `released` with cached `Ready` / `unstarted` metadata after a drained `POST /api/v1/refresh`, even though live Linear had moved the issue to `In Progress` / `started` and provider capacity was available. This fixes the provider-intake recovery path rather than masking the status display.

## Validation

- `node scripts/delegation-guard.mjs`
- `node scripts/spec-guard.mjs --dry-run`
- `npm run test:core -- orchestrator/tests/ProviderIssueHandoffRefreshSerialization.test.ts -t "global capacity is capped"` (1 selected test)
- `npm run test:core -- orchestrator/tests/ProviderIssueHandoffRefreshSerialization.test.ts -t "same-issue occupancy is unreadable"` (2 selected tests)
- `npm run test:core -- orchestrator/tests/ProviderIssueHandoffRefreshSerialization.test.ts` (41 tests)
- Adjacent CO-193 / CO-189 selected coverage (2 tests)
- `npm run build`
- `npm run lint` (passes with 3 pre-existing warnings in `orchestrator/tests/DelegationMcpHealth.test.ts`)
- `npm run test` (355 files, 4930 tests)
- `npm run docs:check`
- `npm run docs:freshness` (4827 docs, 4830 registry entries)
- `npm run repo:stewardship` (5950 tracked files, 0 action-required)
- `node scripts/diff-budget.mjs`
- Manifest-backed forced standalone review after global-cap PR feedback: `review_outcome=clean-success`
- Explicit elegance/minimality pass: no simplification patch needed

Linear: CO-392

## Summary by CodeRabbit

* **New Features**
  * Targeted refresh flow to reconcile stale released-pending-reopen claims when underlying work advances, using a bounded probe to safely refresh claim metadata and resume launches when appropriate.

* **Tests**
  * Expanded tests covering refresh reconciliation, start vs blocked outcomes, fresh-discovery slot behaviour and occupancy edge cases.

* **Documentation**
  * Added PRD, tech spec, action plan, task checklist/records and registry updates documenting design, validation and approvals.

### CODE REVIEW

File: docs/TECH_SPEC-linear-b131706d-e40f-42db-8f29-9f1ddd636187.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Parity / Alignment Matrix
+| Surface | Current Truth | Reference Truth | Target Truth |
+| --- | --- | --- | --- |
+| Provider intake claim | The stale no-run claim keeps cached `Ready/unstarted` after live started truth. | Live issue-by-id or bounded poll truth is newer and started. | The claim records fresh started metadata and either launches/requeues normally or records a concrete skip/recovery reason. |
+| Refresh drain | Accepted refresh can complete without mutating the stale claim. | Refresh completion should mean retained claims were reconciled or skipped with explicit evidence. | Refresh cannot silently preserve the stale started claim shape. |
+| Admission/status | CO STATUS omits the issue while capacity is available. | Provider intake active/admission truth should match live started work after recovery. | Active/running/admission surfaces reflect the recovered claim or the actionable blocker. |
+| Adjacent invariants | CO-193, CO-238, CO-181, and CO-391 own nearby but distinct gaps. | Adjacent lanes keep their existing behavior and tests. | This lane only adds the started no-run released-pending-reopen variant. |
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply docs/TECH_SPEC-linear-b131706d-e40f-42db-8f29-9f1ddd636187.md:31]: @coderabbitai Fixed the MD058 table spacing in docs/TECH_SPEC-linear-b131706d-e40f-42db-8f29-9f1ddd636187.md (commit 2c06994f5). Please re-review/resolve.

File: orchestrator/src/cli/control/providerIssueHandoff.ts [hunk 2]
```diff
@@ -4830,6 +4851,10 @@ export function createProviderIssueHandoffService(
               shouldRefreshReleasedNotActiveMetadataFromBlockerSnapshot
             ) &&
             !shouldBlockPendingReopenFreshDiscovery;
+          const allowDirectIssueById =
+            normallyAllowDirectIssueById || canUseNoRunPendingReopenLiveStartedProbe;
+          const usedNoRunPendingReopenLiveStartedProbe =
+            !normallyAllowDirectIssueById && canUseNoRunPendingReopenLiveStartedProbe;
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/providerIssueHandoff.ts]: Fixed in commit 2c06994f5 by marking the no-run pending-reopen live-started probe as used only from the direct issue-by-id callback, and by adding a poll-snapshot reserved-slot regression. Normal re-review should pick up the new head.

File: docs/PRD-linear-b131706d-e40f-42db-8f29-9f1ddd636187.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+| Provider intake claim | `CO-379` remains `released` with stale `Ready/unstarted` metadata and no run identity. | Live Linear has newer `In Progress/started` truth for the same issue. | Intake refresh/reclaim updates or recovers the claim instead of preserving stale released truth. |
+| Control-host refresh | `POST /api/v1/refresh` is accepted and drains, but the stale claim remains unchanged. | Accepted refresh should reconcile provider-intake truth or expose a concrete recovery blocker. | Drained refresh either repairs/reclaims or records actionable diagnostics for the stale started issue. |
+| CO STATUS | Active/running identifiers omit `CO-379` while capacity is available. | CO STATUS should reflect authoritative provider admission truth for live started work. | Active/running/admission surfaces stop hiding the live started issue after recovery. |
+| Adjacent completed lanes | CO-193 covers Ready/unstarted reclaim, CO-238 covers reopened terminal plain released/not_active, CO-181 covers fair fresh discovery. | Adjacent lanes stay intact without overclaiming this new started/no-run variant. | This lane owns only the started no-run released-pending-reopen refresh gap. |
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply docs/PRD-linear-b131706d-e40f-42db-8f29-9f1ddd636187.md:26]: @coderabbitai Fixed the MD058 table spacing in docs/PRD-linear-b131706d-e40f-42db-8f29-9f1ddd636187.md (commit 2c06994f5). Please re-review/resolve.

File: tasks/specs/linear-b131706d-e40f-42db-8f29-9f1ddd636187.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Parity / Alignment Matrix
+| Surface | Current Truth | Reference Truth | Target Truth |
+| --- | --- | --- | --- |
+| Provider intake claim | The stale no-run claim keeps cached `Ready/unstarted` after live started truth. | Live issue-by-id or bounded poll truth is newer and started. | The claim records fresh started metadata and either launches/requeues normally or records a concrete skip/recovery reason. |
+| Refresh drain | Accepted refresh can complete without mutating the stale claim. | Refresh completion should mean retained claims were reconciled or skipped with explicit evidence. | Refresh cannot silently preserve the stale started claim shape. |
+| Admission/status | CO STATUS omits the issue while capacity is available. | Provider intake active/admission truth should match live started work after recovery. | Active/running/admission surfaces reflect the recovered claim or the actionable blocker. |
+| Adjacent invariants | CO-193, CO-238, CO-181, and CO-391 own nearby but distinct gaps. | Adjacent lanes keep their existing behavior and tests. | This lane only adds the started no-run released-pending-reopen variant. |
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply tasks/specs/linear-b131706d-e40f-42db-8f29-9f1ddd636187.md:31]: @coderabbitai Fixed the MD058 table spacing in tasks/specs/linear-b131706d-e40f-42db-8f29-9f1ddd636187.md (commit 2c06994f5). Please re-review/resolve.

File: orchestrator/src/cli/control/providerIssueHandoff.ts [hunk 5]
```diff
@@ -4804,6 +4809,22 @@ export function createProviderIssueHandoffService(
...
+              latestRun,
+              hasPendingReleaseCancel
+            });
+          const hasOccupiedPollDispatchProviderKey =
+            occupiedPollDispatchProviderKeys.has(claimProviderKey);
+          const canUseNoRunPendingReopenLiveStartedProbe =
+            shouldProbeNoRunPendingReopenLiveStartedTruth &&
+            !hasOccupiedPollDispatchProviderKey &&
+            pollDispatchBudget.remainingGlobalSlots() > 0 &&
+            noRunPendingReopenLiveStartedProbeReads < 1;
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/src/cli/control/providerIssueHandoff.ts:4826]: @coderabbitai Fixed by removing only the global-slot precheck from the no-run pending-reopen live-start probe while preserving same-issue occupancy and one-probe bounds, plus a global-cap max_concurrency regression (commit 410a5a052). Please re-review/resolve.

File: orchestrator/tests/ProviderIssueHandoffRefreshSerialization.test.ts [hunk 6]
```diff
@@ -2075,6 +2075,723 @@ describe('runProviderIssueHandoffRefresh', () => {
...
+        manifestPath: '/tmp/provider-run/co-185-duplicate-live-started-manifest.json'
+      })),
+      resume: vi.fn(async () => undefined)
+    };
+    const resolveTrackedIssue = vi.fn(async () => {
+      throw new Error('same-issue occupied no-run claim should not use direct issue refresh');
+    });
+    const refetchTrackedIssues = vi.fn(async (input: { excludedIssueIds?: string[] }) => {
+      expect(input.excludedIssueIds).toContain('lin-issue-185');
+      return {
```
[the remark this answers is not in the document]
Comment (Kbediako) [reply orchestrator/tests/ProviderIssueHandoffRefreshSerialization.test.ts:2658]: @coderabbitai Fixed by normalizing the optional assertion to `input.excludedIssueIds ?? []` and reran the focused occupancy tests, full refresh serialization suite, and full handoff validation stack (commit 58afaa1e2). Please re-review/resolve.
